src/ai/backend/test/templates/auth/login.py

```diff
@@ -28,16 +24,22 @@ def name(self) -> str:
     async def context(self) -> AsyncIterator[None]:
         test_id = TestIDContext.get_current()
         test_id_str = str(test_id)
-        async with AsyncSession() as session:
+        credential_ctx = LoginCredentialContext.get_current()
+        endpoint_ctx = EndpointContext.get_current()
```

Comment:
This part seems to be an actual value rather than ctx, so the `_ctx` postfix doesn't seem necessary.